app/src/main/java/org/simple/clinic/shortcodesearchresult/ShortCodeSearchResultScreen.kt

```diff
@@ -144,16 +146,20 @@ class ShortCodeSearchResultScreen :
     context.injector<Injector>().inject(this)
   }
 
-  override fun onFinishInflate() {
-    super.onFinishInflate()
-    if (isInEditMode) {
-      return
+  override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+    return super.onCreateView(inflater, container, savedInstanceState).also {
+      patientSearchViewBinding = PatientSearchViewBinding.bind(binding.root)
     }
-    hideKeyboard()
+  }
+
+  override fun onDestroyView() {
+    super.onDestroyView()
+    patientSearchViewBinding = null
+  }
 
-    viewBinding = ScreenShortcodeSearchResultBinding.bind(this)
-    patientSearchViewBinding = PatientSearchViewBinding.bind(viewBinding!!.root)
-    
+  override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+    super.onViewCreated(view, savedInstanceState)
+    (view as ViewGroup).hideKeyboard()
```

Comment:
We don't have to change it now, but we could have used `binding.root` instead of casting it to `ViewGroup`

Reply:
Oh yes, I'll change that in another PR.